public/javascript/gunDrawing.js

```diff
@@ -0,0 +1,29 @@
+const years = document.querySelector(".years");
+const cardDates = document.querySelectorAll(".date");
+
+function winnerDisplay() {
+  cardDates.forEach(date => {
+    const winnerYear = date.innerHTML.slice(-4);
```

Comment:
Shouldn't rely on `innerHTML` for data like this. Use something like data-date.

Reply:
Added data attribute for in template and grabbed it from this line.